lupa/_lupa.pyx

```diff
@@ -426,12 +427,12 @@ cdef class LuaRuntime:
         try:
             check_lua_stack(L, 5)
             lua.lua_newtable(L)
-            # FIXME: how to check for failure?
+            # FIXME: how to check for failure? and nested dict
             for obj in args:
                 if isinstance(obj, dict):
-                    for key, value in obj.iteritems():
-                        py_to_lua(self, L, key)
-                        py_to_lua(self, L, value)
+                    for key, value in obj.iteritems():  # in python3, this is called items
+                        py_to_lua(self, L, key, False, recursive)
+                        py_to_lua(self, L, value, False, recursive)
```

Comment:
Please use keyword arguments if you pass constants like `True` and `False`, since without knowing which argument they refer to, it's unclear what they mean.